src/hooks/useGridKeyboardNavigation/useGridKeyboardNavigation.js

```diff
@@ -0,0 +1,106 @@
+import { useCallback, useLayoutEffect, useState, useContext } from "react";
+import useFullKeyboardListeners from "../useFullKeyboardListeners";
+// import { GridKeyboardNavigationContext } from "../../components/GridKeyboardNavigation/GridKeyboardNavigationContext";
```

Comment:
anything related to the context will be added in the following PR, so it's currently commented out